auth/api/iam/jar_test.go

```diff
@@ -110,8 +111,10 @@ func TestJar_Parse(t *testing.T) {
 	})
 	require.NoError(t, err)
 	token := string(bytes)
+	walletIssuerURL := test.MustParseURL(walletDID.String())
```

Comment:
doesn't this create a DID as URL? Doesn't break the tests but is functionally wrong?

Reply:
Yes, but we don't know the URL beforehand, since it's to be provided by the calling application. With unsolicited requests (OpenID4VP request form verifier), we simply don't know it? I wonder if this shouldn't be the SIOPv2 self-issued URL.

This is actually how it worked in practice before; for did:web DIDs we derived the issuer URL, for other DID methods (`did:jwk` in our case) it's assumed it's the wallet DID, and we just set the DID as issuer URL.